config/routes.rb

```diff
@@ -30,6 +30,12 @@
 
   resources :custom_manifests, only: [:create, :update, :destroy, :show]
 
+  patch "user/counters/:id", to: "user_counters#update"    # Legacy route for GodTools Android v5.7.0-v6.0.0
+  patch "user/me/counters/:id", to: "user_counters#update" # Legacy route for GodTools Android v6.0.1+
```

Comment:
how do these interact with the `scope "user"` block right below it? show the block below this be removed?

Reply:
Oh, yeah, the scope stuff can be removed